java/parquet/src/main/java/sleeper/utils/HadoopConfigurationProvider.java

```diff
@@ -45,6 +58,8 @@ public static Configuration getConfigurationForQueryLambdas(InstanceProperties i
         conf.set("fs.s3a.readahead.range", tableProperties.get(S3A_READAHEAD_RANGE));
         if (System.getenv("AWS_ENDPOINT_URL") != null) {
             setLocalStackConfiguration(conf);
+        } else {
+            conf.set("fs.s3a.aws.credentials.provider", "com.amazonaws.auth.EC2ContainerCredentialsProviderWrapper");
```

Comment:
Should we be setting this here? We weren't setting this on query lambdas before?

Reply:
I have updated this to be the default provider